lib/Service/ActivityService.php

```diff
@@ -187,7 +187,7 @@ private function getMessageFull() {
 			case ShareEvent::REGISTRATION:
 				return $this->userIsActor
 					? $this->trans->t('You registered to poll {pollTitle}')
-					: $this->trans->t('{sharee} registered to poll {pollTitle}');
+					: $this->trans->t('{actor} registered to poll {pollTitle}');
```

Comment:
In this case {sharee} was right to fetch the correct user type, when displaying the user's name